<?xml version="1.0"?>

<launch>
    <node pkg="robotiq_2f_gripper_control" type="Robotiq2FGripperRtuNode.py" name="Robotiq2FGripperRtuNode" args="/dev/ttyUSB1"/>
    <!--node pkg="robotiq_2f_gripper_control" type="Robotiq2FGripperSimpleController.py" name="Robotiq2FGripperSimpleController" output="screen"/-->
</launch>

